mlflow/models/utils.py

```diff
@@ -589,3 +591,94 @@ def validate_schema(data: PyFuncInput, expected_schema: Schema) -> None:
         mlflow.models.validate_schema(input_data, model_signature.inputs)
     """
     _enforce_schema(data, expected_schema)
+
+
+@experimental
+def add_libraries_to_model(model_uri, run_id=None, registered_model_name=None):
+    """
+    This utility function only operates on a model which has been registered to the model registry.
+
+    Given the registered model_uri (models:/<model_name>/<model_version/stage/latest>), it will
```

Comment:
Can we drop the use of the word `will` and instead phrase in the present tense (talk about what the API does, rather than what it will do)

```suggestion
    Given a registered model_uri (models:/<model_name>/<model_version/stage/latest>), it
```